### PR TITLE
Attempt to convert to babel plugin

### LINKED ISSUE #20: Convert to Babel plugin
[Issuehunt badges](https://issuehunt.io/r/sindresorhus/strip-debug/issues/20)

Rocambole, which this project is built on, is no longer maintained.

Would be best to convert this project to a Babel plugin. Then we also get #11 for free.

> There is a $80.00 open bounty on this issue. Add more on [Issuehunt](https://issuehunt.io/r/sindresorhus/strip-debug/issues/20).
> - Checkout the [Issuehunt explorer](https://issuehunt.io/r/sindresorhus/strip-debug/) to discover more funded issues.
> - Need some help from other developers? [Add your repositories](https://issuehunt.io/r/new) on Issuehunt to raise funds.

### PR DESCRIPTION
**Pull request by Elyager**
I converted this repo to a Babel Plugin, is the firs time I do something babel related, let me know any changes or improvements.

I have a identical version of this npm package working on https://www.npmjs.com/package/babel-plugin-noob. I uploaded it because I wanted to test all usage cases, hope you don't mind.

Fixes #20

### CODE REVIEW

**Comment by Elyager** [comment]
Fixes #20

File: package.json [hunk 1]
```diff
@@ -1,19 +1,25 @@
...
 		"node": ">=6"
 	},
 	"scripts": {
-		"test": "xo && ava"
+		"clean": "rm -rf lib",
+		"build": "babel src -d lib",
+		"test": "xo --ignore ./test/fixtures && npm run build && ava test --verbose",
+		"lint-fix": "xo --ignore ./test/fixtures --fix",
+		"test:watch": "npm run build && npm run test -- --watch",
+		"prepublishOnly": "npm run clean && npm run build"
```
Comment (sindresorhus) [inline package.json]: There's no point on transpiling the module code. That just adds a lot of overhead for no gain.

File: package.json [hunk 2]
```diff
@@ -31,17 +37,14 @@
...
-		"rocambole-strip-alert": "^1.0.0",
-		"rocambole-strip-console": "^2.0.0",
-		"rocambole-strip-debugger": "^1.0.0"
-	},
 	"devDependencies": {
 		"ava": "^0.25.0",
+		"babel-cli": "^6.4.5",
+		"babel-core": "^6.26.3",
+		"babel-preset-es2015": "^6.24.1",
+		"babel-preset-stage-0": "^6.24.1",
```
Comment (sindresorhus) [inline package.json]: Use Babel 7

File: test/index.js [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+	const fixtureDir = path.join(fixturesDir, caseName);
+	const actualPath = path.join(fixtureDir, 'actual.js');
+	const actual = transformFileSync(actualPath).code;
+
+	const expected = fs.readFileSync(
+		path.join(fixtureDir, 'expected.js')
+	).toString();
+
+	test(`should ${caseName}`, t => {
+		t.is(trim(actual), trim(expected));
```
Comment (sindresorhus) [inline test/index.js]: Uhm

```suggestion
		t.is(actual.trim(), expected.trim());
```

**Comment by sindresorhus** [comment]
Closing for lack of activity.

**Pull request closed without merging**